### PR TITLE
client/snapctl, store: fix typos

### PR DESCRIPTION
**Pull request by anonymouse64**

### CODE REVIEW

File: store/store_download.go [hunk 1]
```diff
@@ -287,9 +287,9 @@ type TransferSpeedMonitoringWriter struct {
 	measuredWindows int
 }
 
-// NewTransferSpeedMonitoringWriterAndContext returns an io.Writer that measures write speed
-// in measureTimeWindow windows an cancels the operation if minDownloadSeepdBps
-// is not achieved.
+// NewTransferSpeedMonitoringWriterAndContext returns an io.Writer that measures
+// write speed in measureTimeWindow windows and cancels the operation if
+// minDownloadSeepdBps is not achieved.
```
Comment (pedronis) [inline store/store_download.go:292]: minDownloadSpeedBps ?

**Review by pedronis — COMMENTED** [review]
spotted on more typo

File: store/store_download.go (hunk 1 above)
Comment (anonymouse64) [reply store/store_download.go:292]: ah yes good catch

Comment (anonymouse64) [inline store/store_download.go:292]:
```suggestion
// minDownloadSpeedBps is not achieved.
```

**Review by pedronis — APPROVED** [review]
thank you

**Review by mvo5 — APPROVED** [review]
Thank you!

**Pull request merged**